app_flutter/test/navigation_drawer_test.dart

```diff
@@ -114,7 +119,7 @@ void main() {
         ),
       );
 
-      const String skiaPerfText = 'Benchmarks on Skia Perf';
+      const String skiaPerfText = 'Skia Perf';
```

Comment:
+@liyuqian mentioned Skia perf was unclear in https://github.com/flutter/cocoon/pull/687

Reply:
I'm happy to change it back. I was just trying to avoid the word Benchmarks twice but I don't feel strongly about it.